docs/site/Relation-generator.md

```diff
@@ -32,7 +32,8 @@ lb4 relation [options]
 - `--relationType`: Relation type.
 - `--sourceModel`: Source model.
 - `--destinationModel`: Destination model.
-- `--foreignKeyName`: Destination model foreign key name.
+- `--foreignKeyName`: Destination (Source) model foreign key name for HasMany
+  (BelongsTo) relation.
```

Comment:
suggestion: `Destination/Source model foreign key name for HasMany/BelongsTo relation, respectively.` (feel free to ignore)